sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/models/SettingSelector.java

```diff
@@ -49,17 +49,18 @@ public SettingSelector() {
      *
      * <p>
      * Examples:
-     * <ol>
-     * <li>If keys = "*", settings with any key are returned.</li>
-     * <li>If keys = "abc1234", settings with a key equal to "abc1234" are returned.</li>
-     * <li>If keys = "abc*", settings with a key starting with "abc" are returned.</li>
-     * <li>If keys = "*abc*", settings with a key containing "abc" are returned.</li>
-     * </ol>
+     * <ul>
+     * <li>If {@code keyFilter = "*"}, settings with any key are returned.</li>
+     * <li>If {@code keyFilter = "abc1234"}, settings with a key equal to "abc1234" are returned.</li>
+     * <li>If {@code keyFilter = "abc*"}, settings with a key starting with "abc" are returned.</li>
+     * <li>If {@code keyFilter = "*abc*"}, settings with a key containing "abc" are returned.</li>
+     * <li>If {@code keyFilter = "abc,def"}, settings with a key equal to "abc" or "def" are returned.</li>
+     * </ul>
      *
      * @return The expressions to filter ConfigurationSetting keys on.
      */
-    public String[] getKeys() {
-        return keys == null ? new String[0] : CoreUtils.clone(keys);
+    public String getKeyFilter() {
+        return keyFilter == null ? "" : keyFilter;
```

Comment:
Is this correct then?

Reply:
Let me double check with the feature team. The label is optional so it could be null. But the key should be required.

Reply:
In this case, the behaviour is different because you returned an empty set before (nothing in it). But now, you're returning an empty string. Is it possible to have a key with an empty string?

Reply:
You are right. https://github.com/Azure/AppConfiguration/blob/master/docs/REST/kv.md